### ISSUE TITLE
non-www to www forced by default

### ISSUE THREAD

**Issue by khanhquocnguyen** [QROkes/webinoly#33]
I installed my wordpress website by simple command:
`sudo site mysite.com -wp=custom`

But when I check the redirection, the result show me that **non-www always redirect to www** (although in document show that: **By default, Webinoly configures your site to accept both requests in your domain, that is, example.com and www.example.com will be both valid.**).

I tried turn off redirection by running command:
`sudo site mysite.com -force-redirect=off`
But it show _Force-Redirect is already disabled for site mysite.com_

And another issue, If i use **-force-redirect=www** or **-force-redirect=root**, the result is OK but it show more:
```
Force-Redirection to WWW has been successfully enabled!
Job for nginx.service failed because the control process exited with error code.
See "systemctl status nginx.service" and "journalctl -xe" for details.
```

**Comment by QROkes** [comment]
Please, use `sudo nginx -t` and then you can look what's the error in nginx configuration. Tell me what you find.

**Comment by khanhquocnguyen** [comment]
> Please, use `sudo nginx -t` and then you can look what's the error in nginx configuration. Tell me what you find.

I found the reason for issue 2. When you **-force-redirect=www or -force-redirect=root**, some code will be added to nginx configuration file like this:
`letsencrypt/live.... `
But I'm using custom SSL cert so I don't have that directory. It made the issue 2.

**Comment by QROkes** [comment]
Ok. Now I understand!
Are you adding your custom SSL manually or you are using the Webinoly option to add a custom SSL?

**Comment by khanhquocnguyen** [comment]
> Ok. Now I understand!
> Are you adding your custom SSL manually or you are using the Webinoly option to add a custom SSL?

I added custom SSL by Webinoly.

**Comment by QROkes** [comment]
Just for the record, the force-redirect feature uses the SSL cert in the redirection block and fails when it is a custom (non-letsencrypt) cert.

**Scheduled to be fixed in v1.11.1 release.**

Thanks a lot for reporting this issue!

**Comment by QROkes** [comment]
Fixed in Webinoly v1.11.1, released today.
